src/test/resources/all/issues/silver/0586.vpr

```diff
@@ -0,0 +1,17 @@
+field val: Int
+
+define NodeSeg(n) (
+    (let x == (0) in acc(n.val))
+)
+
+method test1(head: Ref)
+{
+	package (NodeSeg(head)) --* true
+	exhale (NodeSeg(head)) --* true
+}
+
+method test2(head: Ref)
+{
+	package (let x == (0) in acc(head.val)) --* true
+	exhale  (let x == (0) in acc(head.val)) --* true
```

Comment:
Should the second `x` here not be e.g. `y` to test the problem you point out in #586?